requirements.txt

```diff
@@ -9,3 +9,5 @@ matplotlib>=3.3.3
 graphviz>=0.13
 seaborn>=0.11.1
 category_encoders>=2.0.0
+pystan
```

Comment:
I left this open for now until we know what versions work with evalml. I can also just use latest versions for now.